android/uhabits-core/src/main/java/org/isoron/uhabits/core/models/memory/MemoryHabitList.java

```diff
@@ -151,6 +151,7 @@ public synchronized void setOrder(@NonNull Order order)
         if (order == BY_NAME) return nameComparator;
         if (order == BY_COLOR) return colorComparator;
         if (order == BY_SCORE) return scoreComparator;
+        if (order == BY_SCORE_ASC) return scoreComparator.reversed();
```

Comment:
I get the following warning in AndroidStudio: 
> Call requires API level 24 (current min is 21): java.util.Comparator#reversed'

Any chance of bumping the minSdkVersion, or shall I just clone the `scoreComparator` and reverse it myself?

Reply:
I wrote out my own reverse comparator in cc4ad53, so we can keep minSdkVersion=21.
Maybe you'd like a TODO so we can simplify this whenever we do bump minSdkVersion.